### DISCUSSION TITLE
Task 3, Problem 6 [About Sampling Frequency]

### DISCUSSION THREAD

**Discussion by AnasAlsharo** [chulminy/CIVE497-CIVE700#154] [Q&A] [answered]
Hi Dr. Yeum, 

In the last problem of task 3 (problem 6) it says "Load `data1.mat` and plot the acceleration signal in a z direction (`zvib`). What is the main frequency of this wave? The sampling frequency is 10.1355 Hz. " However, when I print the data1 file it gives me the following header: 

{'__header__': b'MATLAB 5.0 MAT-file, Platform: PCWIN64, Created on: Mon Dec 30 16:23:32 2019', '__version__': '1.0', '__globals__': [], **'fs': array([[10.11915041]])** is it possible that the sampling frequencies are swapped in the problem or there is something that I miss? 

Thank you!

**Comment by chulminy** [comment, marked as answer]
Yes. You are correct. I will swap the values. 

Chul Min
